columnar_format.ipynb

```diff
@@ -0,0 +1,61 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "e783b4f6-a25f-44b9-8e1f-122759b8d349",
+   "metadata": {},
+   "source": [
+    "# Arrow Columnar Format\n",
+    "\n",
+    "\n",
+    "## Why does Arrow use a Columnar in-memory format?\n",
+    "\n",
+    "When we discuss about Row vs Columnar format we usually discuss about the storage data type. With Arrow we talk about the data structures in memory directly.\n",
```

Comment:
What do you exactly mean here with "storage data type"? 
"data type" already has a specific meaning in context of Arrow as well, but I suppose that is not meant here, which makes it a bit confusing. Something like "... about the way the data is stored"?

Reply:
I would remove two occurences of "about" in the first sentence.